app/trip/components/trip-form.tsx

```diff
@@ -147,8 +151,43 @@ export const TripForm = ({ tripDetails, tags, tripTags }: TripFormProps) => {
         <FormControl isInvalid={!!errors.image_url}>
           <FormLabel>Image</FormLabel>
           <HStack gap={{ base: '20px', md: '34px' }}>
-            <Image alt="" src={tripDetails?.image || imageSrc} width="50%" />
-            <PrimaryButton variant="outline">Select Image </PrimaryButton>
+            {selectedImage ? (
+              <Image
+                alt="Selected Image"
+                src={URL.createObjectURL(selectedImage)}
+                width="50%"
+                objectFit="cover"
+              />
+            ) : (
+              <Image
+                alt="Default Image"
+                src={tripDetails?.image || imageSrc}
+                width="50%"
+                objectFit="cover"
+              />
+            )}
+            <Controller
+              name="uploaded_image_file"
+              control={control}
+              render={({ field: { onChange } }) => (
+                <PrimaryButton variant="outline" as="label">
+                  Select Image
+                  <Input
+                    type="file"
+                    accept="image/*"
```

Comment:
Thank you so much for adding the image upload feature! it's looks really nice! I'm just wondering how you've managed image types and sizes.  I've noticed that it currently accepts various image formats here but I've also noticed that PDFs are not accepted, and there is a limit on the size of images or GIFs that can be inserted. I'm curious about where these settings are configured?

Reply:
I added two validations in zod schema below.
- Only image file
- File size is less than 10MB

https://github.com/tabi-memo/frontend/pull/32/commits/3d3237cae33ee00f1163411a9cd512d4da6e3490